src/main/resources/messages.properties

```diff
@@ -58,3 +58,4 @@ exception.illegal.field.change=You are unable to change your:
 exception.validation=Validation exception
 exception.access-denied.user-data=You do not have access to view the data of another user!
 exception.permission-denied.user-profile=You are not allowed to perform this action
+exception.children-amount=A user can't have more than ${childrenAmount} registered children
```

Comment:
Did you have a possibility to check that ${childrenAmount} param indicates here properly?

Reply:
I checked this manually. I ran tests with `exception.children-amount=A user can't have more than 10 registered children` and they all passed